org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/InvisibleProjectImporter.java

```diff
@@ -105,13 +105,12 @@ public void importToWorkspace(IProgressMonitor monitor) throws OperationCanceled
 				IJavaProject javaProject = JavaCore.create(invisibleProject);
 				ProjectUtils.addSourcePath(sourcePath, subProjectPaths.toArray(new IPath[0]), javaProject);
 				JavaLanguageServerPlugin.logInfo("Successfully created a workspace invisible project " + invisibleProjectName);
+				UpdateClasspathJob.getInstance().updateClasspath(javaProject, preferencesManager.getPreferences().getReferencedLibraries());
```

Comment:
This change has been reverted in new commit https://github.com/eclipse/eclipse.jdt.ls/pull/1305/commits/be30d5d717c679ee0022f5b19a5117531b73d772#diff-c72158b0155b5118904051ab465aad30L108.